src/wechaty_plugin_contrib/contrib/chat_history_plugin/plugin.py

```diff
@@ -1,12 +1,19 @@
 """Chat history plugin"""
 import os
-import aiosqlite
 from typing import Optional, List
 from dataclasses import dataclass, field
 from wechaty_puppet import MessageType  # type: ignore
 from wechaty import Message, get_logger  # type: ignore
 from wechaty.plugin import WechatyPlugin, WechatyPluginOptions  # type: ignore
-
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker, declarative_base
+from sqlalchemy import (  # type: ignore
+    Column,
+    Integer,
+    DateTime,
+    VARCHAR,
+    Text
+)
```

Comment:
you can refer to this pr to disable type hub library linting issue, so that we can remove all of `# type: ignore` to keep code clean.

Reply:
Which PR? It seems that mypy cannot pass without `# type: ignore`, thanks.

Reply:
Oh, I have updated the link of pr, please check it out.